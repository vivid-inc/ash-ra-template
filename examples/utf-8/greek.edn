{Α  "άλφα"
 Β "βήτα"
 Γ "γάμμα"}
