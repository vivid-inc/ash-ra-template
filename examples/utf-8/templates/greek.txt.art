Τα πρώτα γράμματα του αλφαβήτου είναι {{ (clojure.string/join ", " [Α Β Γ]) }}.
