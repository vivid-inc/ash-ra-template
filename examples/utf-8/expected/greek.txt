Τα πρώτα γράμματα του αλφαβήτου είναι άλφα, βήτα, γάμμα.
